src/arrayMethodJoin.js

```diff
@@ -4,8 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinedToString = '';
+
+    for (let i = 0; i < this.length; i++) {
+      let itemOfThis = this[i];
+
+      if (itemOfThis === null || itemOfThis === undefined) {
+        itemOfThis = '';
+      }
+
+      joinedToString += itemOfThis;
```

Comment:
you can invert your check so you'd only do one thing
```suggestion
      if (inverted check here) {
        joinedToString += itemOfThis;
      }
```